programs/drift/src/validation/perp_market.rs

```diff
@@ -10,6 +10,16 @@ use solana_program::msg;
 
 #[allow(clippy::comparison_chain)]
 pub fn validate_perp_market(market: &PerpMarket) -> DriftResult {
+    let (_standardized_delta_base, _remainder_base_asset_amount) =
```

Comment:
_remainder_base_asset_amount should be remainder_base_asset_amount since it's used below